webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -636,6 +637,15 @@ class DSCTechniqueWriter(
         <FILE name={s"RUDDER_CONFIGURATION_REPOSITORY/${computeTechniqueFilePath(technique)}"}>
           <INCLUDED>true</INCLUDED>
         </FILE>
+        { for {
```

Comment:
cn you put the open `{` on previoux line? Sometimes, when we open it on a new line, scalac parser just ignore the code produced. It can depend on scala version, which makes it hard to always notice. Having it on the same line prevent that. 
And in miror, put the last `}` just before `<FILES>.